### PR TITLE
Activities list activity: fix hsl computation, fix header color

### LINKED ISSUE #1422: Font problem on bright primary color
Use theming color: `#DFFFF4`

❌ Font is white instead of black
❌ Some activity items are blue instead of the color
🔧 When the theming color is too bright, we should use black as a font in activities

> bildschirmfoto vom 2017-08-24 15-01-55

cc @tobiasKaminsky @AndyScherzinger

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Computation of hsl was wrong, therefore always a white font was shown.

Fix #1422

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
👍 fine!

File: src/main/java/com/owncloud/android/ui/activity/ActivitiesListActivity.java [hunk 1]
```diff
@@ -134,7 +135,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // setup drawer
         setupDrawer(R.id.nav_activity);
-        getSupportActionBar().setTitle(getString(R.string.drawer_item_activities));
+        ActionBar actionBar = getSupportActionBar();
+        ThemeUtils.setColoredTitle(actionBar, getString(R.string.drawer_item_activities));
```
Comment (AndyScherzinger) [inline src/main/java/com/owncloud/android/ui/activity/ActivitiesListActivity.java]: Please add a null check for actionBar

**Comment by AndyScherzinger** [comment]
👍 fine!

**Comment by mario** [comment]
👍

**Pull request merged**